lib/shim/webframework-shim.js

```diff
@@ -117,7 +116,6 @@ WebFrameworkShim.prototype.errorHandled = errorHandled
 WebFrameworkShim.prototype.setErrorPredicate = setErrorPredicate
 WebFrameworkShim.prototype.setResponsePredicate = setResponsePredicate
 WebFrameworkShim.prototype.savePossibleTransactionName = savePossibleTransactionName
-WebFrameworkShim.prototype.captureUrlParams = captureUrlParams
```

Comment:
I would also include in release notes that you removed this method.  Not sure of alternative for users who may have been using this

Reply:
I can re-add it, I totally spaced that this was external facing as well, I just noticed we weren't using it internally at all and naively removed it

Reply:
I think it's ok you remove it